brainscore_language/submission/endpoints.py

```diff
@@ -41,11 +51,20 @@ def send_email_to_submitter(uid: int, domain: str, pr_number: str,
 
 if __name__ == '__main__':
     parser = make_argparser()
+    parser.add_argument('--fn', type=str, nargs='?', default='run_scoring',
```

Comment:
Is there a reason to add this here instead of including it directly in `make_argparser()`?

Reply:
Good point, especially since the same functionality will apply to other domains. I moved to `make_argparser` in Core [12fcfbc](https://github.com/brain-score/core/pull/40/commits/12fcfbce5470b28f9e79fcb385a7b45416efd830) and removed it from Language in [6cfd50c](https://github.com/brain-score/language/pull/204/commits/6cfd50ceab6909706e44f2344a95a0345fb11200).